STACpopulator/populator_base.py

```diff
@@ -142,23 +146,39 @@ def create_stac_collection(self) -> dict[str, Any]:
     def publish_stac_collection(self, collection_data: dict[str, Any]) -> None:
         post_stac_collection(self.stac_host, collection_data, self.update, session=self._session)
 
+    def configure_app_logging(self, log_debug) -> None:
+        """Configure the logger for the App."""
+        # generating the log file name
+        implementation_name = type(self).__name__
+        fname = f"{implementation_name}_log_{datetime.strftime(datetime.now(), '%Y%m%d-%H%M%S')}.jsonl"
+        log_level = "DEBUG" if log_debug else "INFO"
+        setup_logging(fname, log_level)
```

Comment:
I suggest using `logging.INFO`/`logging.DEBUG` and use `logging.getLevelName` in the function for the configuration.

Reply:
Sure... 🤷‍♂️ Don't see any obvious or materially significant benefit from this..

Reply:
Avoid hardcoded string as much as possible to limit side-effects or having to remember (which we won't) all the places where does hardcoded string impact the code. Using predefined references makes the intention clearer.

Reply:
Well.... not quite. What you are saying is true, but only for strings that are hardcode with some information that is custom to the application. But, in this case, "DEBUG" or "INFO" are not going to mean anything else, like ever (or at least in our lifetimes :D). And if one ever has to change debugging levels in the app, searching for all instances of `logging.DEBUG` (and so on) is no different from searching for all instances of `"DEBUG"` (and so on). So, I think this is one of those things where the hardcoded string doesn't matter. However, it's changed in the code to use the more verbose form.

Reply:
Given this use case, fair enough to leave as is.